src/types/common/enums/common.enums.ts

```diff
@@ -67,3 +67,7 @@ export enum StatusEnum {
   Active = 'active',
   Closed = 'closed'
 }
+export enum DrawerAnchorEnum {
```

Comment:
IconPositionEnum does the same, create common enum for both

Reply:
It will be PositionEnum, I told him

Reply:
In the future we can all top and bottom